nms/app/alerts/cwfAlerts.js

```diff
@@ -14,7 +14,7 @@
  * @format
  */
 
-import type {prom_alert_config} from '../generated/MagmaAPIBindings';
+import type {prom_alert_config} from '../../generated/MagmaAPIBindings';
```

Comment:
`sync.js` and everything else in `/alerts` is only imported from `alerts/routes.js`. Therefore everything should be moved to `server/alerts`.